x-pack/filebeat/module/o365/audit/test/22-yammer.log-expected.json

```diff
@@ -0,0 +1,110 @@
+[
+    {
+        "@timestamp": "2020-02-28T09:42:45.000Z",
+        "client.address": "79.159.10.151",
+        "client.ip": "79.159.10.151",
+        "event.action": "GroupCreation",
```

Comment:
groupcreate would probably be event.category = iam, event.type = ["create", "group"]

Reply:
Thanks for the suggestion. I wasn't aware of this new category value.